xbmc/settings/dialogs/GUIDialogSettingsBase.cpp

```diff
@@ -521,13 +521,20 @@ std::set<std::string> CGUIDialogSettingsBase::CreateSettings()
     const CSettingControlTitle *title = dynamic_cast<CSettingControlTitle *>((*groupIt)->GetControl());
     bool hideSeparator = title ? title->IsSeparatorHidden() : false;
```

Comment:
I reread your comment above and now understood your usecase now. But it really doesn't make sense to show the separator above the first item. If a container/list needs a visual separation of some other elment, then this separation should be added by the skin. Just think about a case where you would misuse the separator from here but the list is too long so that the container needs scrolling. As soon as you scroll your separator would be gone. So as I said, separation of a grouplist/container to some other element must not be done by misusing these separators but independently.